portstyle.css

```diff
@@ -7,32 +7,33 @@ body {
 	margin: 0;
 }
 
+.fontStyleHead {
+	font-family: Arial, sans-serif;
+	font-size: 1.2rem;
+	color: white;
+	text-decoration: none;
+}
+
```

Comment:
Text decoration none isn't changing anything in your current structure, if you didn't have buttons around your link text you'd have a problem with underlines. But you can probably just apply text-decoration: none to your navbar to fix this.

Reply:
I actually did try and that and couldn't get it to work which is why I resorted to buttons. I have removed the text-decoration: on line 14.

Reply:
If you do decide to remove the buttons, 
.navigationBar a {
    text-decoration: none; 
}
will remove the underlines